### ISSUE TITLE
Event Listener for NavBar (Enter)

### ISSUE THREAD

**Issue by mjleavell** [nss-evening-cohort-8/chatty-spatulas#19]
# User Story
As a user, I want my text input message to be displayed on the page that includes timestamp, username (bold), and message.. 

# Acceptance Criteria
* When I type a message and click enter, it should be displayed in the message-output div to include the username in bold, message, and timestamp.

# Technical Notes
* import printToDom from util.js
* create enterMsgEvent event listener (keypress) to text input on navbarEvents.js that calls messageBuilder function

**Comment by MCrank** [comment]
```html
<div id="messageId">
    <h5>9/10/2018 12:00:00<strong>Marco</strong></h5>
    <p>Four Score and Seven Years Ago</p>
    <input type="button" value="delete">
    <input type="button" value="edit">
</div>

**Comment by mjleavell** [comment]
![image](https://user-images.githubusercontent.com/42177496/46703085-40d11f00-cbeb-11e8-89f8-e58b6f064993.png)

**Comment by mjleavell** [comment]
![image](https://user-images.githubusercontent.com/42177496/46705845-fbb2ea00-cbf6-11e8-86ce-ea82f2b86b0d.png)

**Comment by MCrank** [comment]
Merged PR #45
